ai2thor/tests/test_unity.py

```diff
@@ -1136,6 +1147,10 @@ def test_change_resolution_image_synthesis(fifo_controller):
     assert event.depth_frame.shape == (300, 300)
     assert event.instance_segmentation_frame.shape == (300, 300, 3)
     assert event.semantic_segmentation_frame.shape == (300, 300, 3)
+    print("is none? {0} is none other {1} ".format( event.depth_frame is None, first_depth_frame is None))
+    save_image("depth_after_resolution_change_300_300.png", depth_to_gray_rgb(event.depth_frame))
+    save_image("before_after_resolution_change_300_300.png", depth_to_gray_rgb(first_depth_frame))
+
```

Comment:
Is this primarily for after-the-fact debugging on the build machine?

Reply:
Yes,  will implement a better system for this unity tests and the ones in the Rendering Tests PR